### DISCUSSION TITLE
Jyers install Ender 3 v2 Neo

### DISCUSSION THREAD

**Discussion by Savel78** [Jyers/Marlin#2082] [Q&A]
Hello

i hope my english is good to understand. Let me try.
I have the Ender 3 v2 neo Printer with Bl touch and a 4.2.2 board. I want to install the jyers Firmware but my Installation stuck with the Creality Screen. 
I read about a firmware (Professional firmware namend Ender 3v2 Blt 422. That worked for me. But i want the Jyers Firmware. What can i do to install that firmware?

Thx for help

**Comment by RV-from-be** [comment]
@Savel78 
Just go to https://discord.gg/NS4zYd23dX. You can find an advanced firmware based on Jyersui with a lot of functionality (more than the pseudo Proui)....

**Comment by williamnauen** [comment]
Hi is anyone looking at making the nov professional release compatible with the ender 3 max neo?Thanks advance WilliamSent from my iPhoneOn 28 Mar 2023, at 13:17, RV-from-be ***@***.***> wrote:﻿
@Savel78
Just go to https://discord.gg/NS4zYd23dX. You can find an advanced firmware based on Jyersui with a lot of functionality (more than the pseudo Proui)....

**Comment by Savel78** [comment]
In english whould bei fine..Gesendet von Outlook für Android
